sdk/servicelinker/Azure.ResourceManager.ServiceLinker/tests/Tests/WebAppKeyVaultConnectionTests.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.ResourceManager.AppService;
+using Azure.ResourceManager.KeyVault;
+using Azure.ResourceManager.KeyVault.Models;
+using Azure.ResourceManager.Resources;
+using Azure.ResourceManager.ServiceLinker.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.ServiceLinker.Tests.Tests
+{
+    [TestFixture]
+    public class WebAppKeyVaultConnectionTests : ServiceLinkerTestBase
+    {
+        public WebAppKeyVaultConnectionTests() : base(true)
+        {
+        }
+
+        [SetUp]
+        public async Task Init()
+        {
+            await InitializeUserTokenClients();
+        }
+
+        [TestCase]
+        public async Task WebAppKeyVaultConnectionCRUD()
```

Comment:
In general, we should test List

Reply:
Added `List service linker` in CRUD test